### DISCUSSION TITLE
Wrap/Unwrap lines in view mode

### DISCUSSION THREAD

**Discussion by ghanan** [damionx7/Zettel-Notes-Documentation#161] [Feature Requests]
not only in edit mode.

**Comment by damionx7** [comment]
Hey. This option already exists. Please check the attached screenshot.

![Screenshot_2024-10-07-21-59-56-86_377fe8767025e4fa6dddb62d6df8f716.jpg](https://github.com/user-attachments/assets/0403b32b-0900-4c40-9fdd-2c695d6faf43)

**Reply by ghanan** [reply]
I mean to change while your are viewing the note, not as an configuration option.

**Reply by damionx7** [reply]
So, like an option in toolbar during view mode.
